create-project-async/src/project.py

```diff
@@ -79,3 +93,18 @@ def __add_headers(self, key, value):
             self.headers[key] = value
 
         return self.headers
+
+    def __sort_possible_extra_files_last(self, filepaths):
+        # Sort file paths ending with .json or .txt to be at the end
+        filepaths.sort(key=lambda x: (x.endswith('.json') or x.endswith('.txt'), x))
```

Comment:
ini jadi semua file `txt` dan `json` bakal return (True, <filename>) 

file-file lain bakal (False, <filename>)

gitu ya?

Reply:
kok bisa key nya tuple 🤔

Reply:
aku masih ga paham sih gimana python bisa sorting by comparing tuples, but it seem to work
![image](https://github.com/datasaur-ai/api-client-example/assets/89899945/a9797554-849a-4ce1-96e4-b3bbdc5214f4)

Reply:
aku juga masih blm paham detail, mostly trial and error bersama chatgpt 😅